jdbc/jooq/gensrc/main/java/example/springdata/jdbc/basics/simpleentity/domain/DefaultCatalog.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * This file is generated by jOOQ.
```

Comment:
Generated files shouldn't be checked into version control. Obviously applies to all generated files.

Reply:
I have used [this example](https://github.com/spring-projects/spring-boot/tree/master/spring-boot-samples/spring-boot-sample-jooq) as a reference, so I included the generated sources. I will remove these because I also think it makes no sense to include them.